client/src/InsertVideo.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Delete from "./clickables/Delete";
+
+export default function InsertVideo({video}) {
+    const src = video.url.replace("watch?v=", "embed/")
```

Comment:
Nice way of embedding a YouTube video without having to extract an id from a url!
How did you come up with this solution?

Reply:
I realize embedding the url is not working and I checked samples of embedded YouTube url and realised the difference.